presidio-anonymizer/Dockerfile

```diff
@@ -10,5 +10,5 @@ RUN pipenv sync
 
 COPY . /usr/bin/${NAME}/
 
-EXPOSE ${FLASK_RUN_PORT}
-CMD pipenv run flask run --host 0.0.0.0
\ No newline at end of file
+EXPOSE ${PORT}
+CMD pipenv run python app.py --host 0.0.0.0
```

Comment:
since we are going to change this to flask run, please add a TODO comment with the relevant user id/bug

Reply:
Done